src/components/Inventory/withdraw/flow/SelectPurposeForm.tsx

```diff
@@ -72,32 +68,30 @@ export default function SelectPurposeForm(props: SelectPurposeFormProps): JSX.El
   };
 
   useEffect(() => {
-    const batchId = batchIds[0];
-    const fetchBatch = async () => {
-      const response = await getBatch(Number(batchId));
-      if (response.requestSucceeded && response.batch) {
-        setBatch(response.batch);
-        setRecord({
+    const fetchBatches = () => {
+      const withdrawalBatches: NurseryWithdrawal[] = [];
+      batches.forEach((batch, index) => {
+        const withdrawalBatch: NurseryWithdrawal = {
+          id: -1,
           purpose: 'Out Plant',
```

Comment:
assuming you will change this to something else and hide outplant for contributors

Reply:
Yes, will do it in separate PR